.github/workflows/cd.yml

```
@@ -0,0 +1,46 @@
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    environment:
+      name: staging
+    concurrency:
+      group: cd-staging-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Resolve tag for main
+        id: meta
+        run: echo "TAG=latest" >> $GITHUB_OUTPUT
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:1:22: Double quote to prevent globbing and word splitting

(shellcheck)